public/editor/scripts/main.js

```diff
@@ -47,6 +47,7 @@ require(["jquery", "bowser", "analytics"], function($, bowser, analytics) {
   function onError(err) {
     console.error("[Bramble Error]", err);
     $("#spinner-container").addClass("loading-error");
+    analytics.event({ category : analytics.eventCategories.TROUBLESHOOTING, action : "Editor loading error (black Screen)" });
```

Comment:
To be consistent, let's change to `Black screen` (capital `B`) since you do `Green screen` below.

Also, what do you think about including the `err` in your action?  @gideonthomas is this a good or bad idea?  It would at least give us a fast look at what's happening